linera-base/src/crypto.rs

```diff
@@ -300,24 +299,22 @@ impl TryFrom<&[u8]> for CryptoHash {
     type Error = CryptoError;
 
     fn try_from(value: &[u8]) -> Result<Self, Self::Error> {
-        if value.len() != HasherOutputSize::to_usize() {
+        if value.len() != B256::len_bytes() {
             return Err(CryptoError::IncorrectHashSize(value.len()));
         }
-        let mut bytes = HasherOutput::default();
-        bytes.copy_from_slice(value);
-        Ok(Self(bytes))
+        Ok(Self(B256::from_slice(value)))
     }
 }
 
 impl From<[u64; 4]> for CryptoHash {
     fn from(integers: [u64; 4]) -> Self {
-        CryptoHash(u64_array_to_le_bytes(integers).into())
+        CryptoHash(u64_array_to_be_bytes(integers).into())
     }
 }
 
 impl From<CryptoHash> for [u64; 4] {
     fn from(crypto_hash: CryptoHash) -> Self {
-        le_bytes_to_u64_array(&crypto_hash.0)
+        be_bytes_to_u64_array(crypto_hash.0.as_ref())
```

Comment:
Is that so that a BCS-encoded `CryptoHash` has the same sequence of bytes as the `B256`?

Reply:
Keccak256 internally is big-endian so I thought we should be consistent with that.